run_test.sh

```diff
@@ -1319,7 +1319,7 @@ echo 'Running autoinstall with failing test module and test module with dependen
 run_with_expected_error 11 dkms autoinstall -k "${KERNEL_VER}" << EOF
 
 Building module:
-Cleaning build area...(bad exit status: 2)
+Cleaning build area...
```

Comment:
The (2x) bad exit status should squashed with another patch, right?

Reply:
No. This patch fixes malformed Makefiles indented with spaces instead of tabs.